Uppsala/UppEt9.xml

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt9">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        
+            <fileDesc>
+            <titleStmt>
+                <title xml:id="t1">
+                    Wəddāse ʾAmlāk, ʾAstabqʷəʿot ʾənta za-Səmʿon ʿAmdāwi
+                </title>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0.
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0356Upp"/>
+                        <collection>O Etiop.</collection>
+                        <idno>O Etiop. 9</idno>
+                        <altIdentifier><idno>Löfgren 15</idno></altIdentifier>
+                        <altIdentifier><idno>Zetterstéen IX</idno></altIdentifier>
+                        <altIdentifier><idno>Äthiop. Ms. 2</idno></altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="5" to="172r"/>
+                            <title type="complete"><title type="complete" ref="LIT2505Weddas"/> and 
```

Comment:
I am not sure if the title inside title encoding works - if there is a precedent, great, otherwise I would suggest to write out the content `<title type="complete">Wəddāse ʾAmlāk and ʾAstabqʷəʿot ʾənta za-Səmʿon ʿAmdāwi</title>`